requirements.txt

```
@@ -0,0 +1,2 @@
+ConfigArgParse
+requests
```

I had this in #5 

```suggestion
requests~=2.12
```

Looking at it know it seems that might be to modern for CentOS 7. Independent of how you decide, I'll probably looking into downgrading all of these to what is available for the centos packages at the time we build them.